src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -177,5 +209,338 @@ private void ThrowIfDisposed()
                 throw new ObjectDisposedException(null);
             }
         }
+
+        private void InitializeMetricsClients()
+        {
+            if (_metricsReceiver == null)
+            {
+                _metricsReceiver = _messagingProvider.CreateMessageReceiver(_entityPath, _serviceBusAccount.ConnectionString);

```

Comment:
`_metricsReceiver` will be equal `_receiver` (if sessions is not enabled). Why do we need both?